.github/workflows/renovate.yml

```diff
@@ -0,0 +1,25 @@
+name: Renovate
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Override default log level"
+        required: false
+        default: "info"
+        type: string
+      overrideSchedule:
+        description: "Override all schedules"
+        required: false
+        default: "false"
+        type: string
+  # Run twice in the early morning (UTC) for initial and follow up steps (create pull request and merge)
+  schedule:
+    - cron: '30 4,6 * * *'
```

Comment:
@macedogm I notice that the config is getting set so that every project runs at `30 4,6 * * *`. Will this create a thundering herd problem if all repos use the same cron config? Should this be spread out?

Reply:
It might create, but since multiple teams own the repos, I believe this shouldn't be an issue, because it will not put the pressure in only one team. Besides, they can still review/merge at their own pace.

If you believe that this will be a big issue, we can change the schedule manually as we open the PRs.